staging/src/k8s.io/mount-utils/mount_linux.go

```diff
@@ -663,6 +674,19 @@ func forceUmount(path string) error {
 	out, cmderr := cmd.CombinedOutput()
 
 	if cmderr != nil {
+		return fmt.Errorf("unmount failed: %v\nUnmounting arguments: -f %s\nOutput: %s", cmderr, path, string(out))
+	}
+	return nil
+}
+
+func quietUmount(path string) error {
+	cmd := exec.Command("umount", path)
```

Comment:
is `-q` missing here?

Reply:
Yes, my bad, I was messing around with PR and CI last minute.

 I have to rename the function and change the approach (maybe I should open a new PR?) because

- umount -q is not available in all versions of util-linux, as u pointed out below.
- even where it is available, all it does is prevent the "not mounted" error message from printing, the exit code is still >0 (32?)
```
$ sudo umount asdf
umount: asdf: not mounted.
$ echo $?
32
$ sudo umount -q asdf
$ echo $?
32
```

so actually I think the change should be `check for 'not mounted' error` , not `use umount -q`

Reply:
i've reworked the PR so we `check for 'not mounted' error `,  instead of relying on `umount -q`